ansible/group_vars/nimbus.holesky.yml

```
@@ -92,7 +92,12 @@ beacon_node_validator_monitor_details: >-
   {{  (node.public_api is not defined or not node.public_api)
   and (node.end is defined and (node.end - node.start) < 10) }}
 # Execution layer Enginer API
-beacon_node_exec_layer_urls: ['http://localhost:{{ exec_layer_authrpc_port }}']
+beacon_node_exec_layer_urls: >-
+  {%- if nodes_layout[inventory_hostname][idx].get('rpc_snooper', false) -%}
+    ['http://localhost:{{ rpc_snooper_cont_port }}']
+  {%- else -%}
+    ['http://localhost:{{ exec_layer_authrpc_port }}']
+  {%- endif -%}
```

How about make it simpler, and just use `node` object from the loop:
```suggestion
beacon_node_exec_layer_urls: ['http://localhost:{{ rpc_snooper_cont_port if node.get("rpc_snooper", false) else exec_layer_authrpc_port }}']
```

---

Always forget this, lovely, thank you